### ISSUE TITLE
Problem running a demo on Windows 10 using conda

### ISSUE THREAD

**Issue by RobinLbt** [mingfeisun/DeepMimic_mujoco#2]
I've tried to run a DeepMimic_mujoco demo on Windows 10 using conda with python 3.6
Here is what happens when I try to run env _torque_test.py

> (base) C:\Users\Utilisateur\Desktop\DeepMimic_mujoco-master\src>py env_torque_test.py
running build_ext
building 'mujoco_py.cymj' extension
Traceback (most recent call last):
  File "env_torque_test.py", line 2, in <module>
    from dp_env_v3 import DPEnv
  File "C:\Users\Utilisateur\Desktop\DeepMimic_mujoco-master\src\dp_env_v3.py", line 10, in <module>
    from mujoco_py import load_model_from_xml, MjSim, MjViewer
  File "c:\users\utilisateur\desktop\mujoco-py-1.50.1.0\mujoco_py\__init__.py", line 1, in <module>
    from mujoco_py.builder import cymj, ignore_mujoco_warnings, functions, MujocoException
  File "c:\users\utilisateur\desktop\mujoco-py-1.50.1.0\mujoco_py\builder.py", line 283, in <module>
    cymj = load_cython_ext(mjpro_path)
  File "c:\users\utilisateur\desktop\mujoco-py-1.50.1.0\mujoco_py\builder.py", line 54, in load_cython_ext
    cext_so_path = builder.build()
  File "c:\users\utilisateur\desktop\mujoco-py-1.50.1.0\mujoco_py\builder.py", line 131, in build
    dist.run_commands()
  File "C:\Users\Utilisateur\Anaconda3\lib\distutils\dist.py", line 955, in run_commands
    self.run_command(cmd)
  File "C:\Users\Utilisateur\Anaconda3\lib\distutils\dist.py", line 974, in run_command
    cmd_obj.run()
  File "C:\Users\Utilisateur\Anaconda3\lib\site-packages\Cython\Distutils\old_build_ext.py", line 186, in run
    _build_ext.build_ext.run(self)
  File "C:\Users\Utilisateur\Anaconda3\lib\distutils\command\build_ext.py", line 339, in run
    self.build_extensions()
  File "c:\users\utilisateur\desktop\mujoco-py-1.50.1.0\mujoco_py\builder.py", line 75, in build_extensions
    build_ext.build_extensions(self)
  File "C:\Users\Utilisateur\Anaconda3\lib\site-packages\Cython\Distutils\old_build_ext.py", line 194, in build_extensions
    self.build_extension(ext)
  File "C:\Users\Utilisateur\Anaconda3\lib\distutils\command\build_ext.py", line 533, in build_extension
    depends=ext.depends)
  File "C:\Users\Utilisateur\Anaconda3\lib\distutils\_msvccompiler.py", line 304, in compile
    self.initialize()
  File "C:\Users\Utilisateur\Anaconda3\lib\distutils\_msvccompiler.py", line 197, in initialize
    vc_env = _get_vc_env(plat_spec)
  File "C:\Users\Utilisateur\Anaconda3\lib\distutils\_msvccompiler.py", line 85, in _get_vc_env
    raise DistutilsPlatformError("Unable to find vcvarsall.bat")
distutils.errors.DistutilsPlatformError: Unable to find vcvarsall.bat

Any help would be great !

**Comment by mingfeisun** [comment]
Hi, 
Thanks for raising this issue. The DeepMimic_mujoco was developed on Ubuntu 16.04. So there might be compatibility issues running on Windows. 

I guess this error is caused by the incorrect installation of mujoco_py. I googled this error and find the following answers: 
* [How to deal with the pain of “unable to find vcvarsall.bat”](https://devblogs.microsoft.com/python/unable-to-find-vcvarsall-bat/)
* [error: Unable to find vcvarsall.bat](https://stackoverflow.com/questions/2817869/error-unable-to-find-vcvarsall-bat)

Hope these could help. One more thing, could you test 'import mujoco_py' alone to see whether the mujoco_py and mujoco150 have been successfully installed.

**Comment by RobinLbt** [comment]
I've tried import mujoco_py and it wasn't been successfully installed. The problem was from here

![Capture](https://user-images.githubusercontent.com/25883677/57979899-d6574b80-7a23-11e9-9cad-8029a2388a93.PNG)

 I've reinstall it and test it and now it's installed successfully. ( Examples of mujoco-py works)
Then I've tried to run py play_mocap.py again and my first issue with vcvarsall.bat is gone but nothing happens and mujoco doesn't launch.

![Capture3](https://user-images.githubusercontent.com/25883677/57979905-e53dfe00-7a23-11e9-80cc-2b7175c11b59.PNG)
